spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
+  let!(:tag1a) { FactoryGirl.create(:classification, :name => 'tag1a') }
+  let!(:tag2a) { FactoryGirl.create(:classification, :name => 'tag2a') }
+  let!(:tag3a) { FactoryGirl.create(:classification, :name => 'tag3a') }
+  let!(:folder1a) do
+    f1 = FactoryGirl.create(:classification, :name => 'folder1a', :show => true)
+    f1.entries.push(tag1a)
+    f1.entries.push(tag2a)
+    f1.entries.push(tag3a)
+    f1
+  end
+  let!(:tag1b) { FactoryGirl.create(:tenant, :name => 'tag1b') }
+  let!(:tag2b) { FactoryGirl.create(:tenant, :name => 'tag2b') }
+  let!(:tag3b) { FactoryGirl.create(:tenant, :name => 'tag3b') }
+  let!(:tree_name) { :alert_profile_obj }
```

Comment:
I deleted this, because it was used only once, other names were already with symbol. But i disagree with this approach, its better to define the name in one place if you are using it more frequently, otherwise you are using magic constant which is not good programming practice.